ibft/ibft.go

```diff
@@ -95,13 +88,20 @@ func (i *ibftImpl) StartInstance(opts StartOptions) (bool, int, []byte) {
 	}
 
 	newInstance := NewInstance(InstanceOptions{
-		Logger:         opts.Logger,
-		Me:             i.me,
+		Logger: opts.Logger,
+		Me: &proto.Node{
+			IbftId: opts.Duty.NodeID,
```

Comment:
is this a preparation for multi validator support?

Reply:
@nivBlox

Reply:
@alonmuroch Yes, we have other issue for that. #55